configtx/application.go

```diff
@@ -216,10 +228,21 @@ func (a *ApplicationOrg) Policies() (map[string]Policy, error) {
 	return getPolicies(a.orgGroup.Policies)
 }
 
+// SetModPolicy sets the specified modification policy for the application organization group
```

Comment:
Godoc uses should use complete sentences with a period. (Here and at all of the other spots where doc was added.)